Fabric.Authorization.IntegrationTests/Modules/RolesTests.cs

```diff
@@ -233,5 +239,208 @@ public void TestDeleteRole_Fail(string id)
 
             Assert.Equal(HttpStatusCode.NotFound, delete.StatusCode);
         }
+
+        [Fact]
+        public void Test_DeletePermissionFromRole_PermissionDoesntExist_NotFoundException()
```

Comment:
Do we have a test that covers the scenario "delete a permission from role but the role doesn't contain the permission" (permission **does exist in the system, just not on the role)**?

Reply:
That is essentially what this is doing. We dont check the existence of the permission in the system on delete because we check that when its added to a role. So this check just ensures that the permission being deleted exists on the role